contrib/libxo/libxo/libxo.c

```diff
@@ -274,31 +274,6 @@ struct xo_handle_s {
     void *xo_private;		/* Private data for external encoders */
 };
 
-#ifdef __CHERI_PURE_CAPABILITY__
```

Comment:
Maybe extend the revert commit message with something like:

This hack was useful for debugging prior to a working GDB, but it does not work on Morello since va_list is not a single pointer.
Moreover, most libxo-using programs have been fixed and we can now use GDB to debug these failures.